src/Menu/BackendMenuBuilder.php

```diff
@@ -37,8 +37,12 @@ final class BackendMenuBuilder implements BackendMenuBuilderInterface
     /** @var ContentExtension */
     private $contentExtension;
 
+    /** @var ExtensionBackendMenuInterface[] */
+    private $extensionMenu;
+
     public function __construct(
         FactoryInterface $menuFactory,
+        iterable $extensionMenu,
```

Comment:
maybe use plural $extensionMenus to indicate it is a list?

Reply:
correct!